docs/guides/cloud/smart-orchestration/parallelization.mdx

```diff
@@ -223,7 +223,7 @@ runs can be utilized independently of Cypress parallelization.
 
 :::info
 
-## Cross Browser Testing
+<strong>Cross Browser Testing</strong>
```

Comment:
was there a reason to using html here to make it bold instead of markdown?

Reply:
Yeah it a little hacky but using the markdown syntax adds in these admonitions throws additional line height/padding below the title and makes the overall height taller. I think some some of these were done with html in the old site and when we ported over this became clear so we just went with this pattern to keep things more snug until the redesign is accounted for.